pkgs/development/libraries/libgweather/default.nix

```diff
@@ -74,6 +74,8 @@ stdenv.mkDerivation rec {
     updateScript = gnome.updateScript {
       packageName = pname;
       versionPolicy = "odd-unstable";
+      # Version 40.alpha preceded version 4.0.
+      freeze = "40.alpha";
```

Comment:
_Only to check my understanding..._

This means that only versions smaller than `40.alpha` will be considered, right?

And if `freeze` is `true` (instead of a string), only versions smaller than the upper bound (automatically calculated as 1 plus the minor version) will be considered, right?

Reply:
> This means that only versions smaller than `40.alpha` will be considered, right?

That is correct, the upper bound is open/exclusive.

> And if `freeze` is `true` (instead of a string), only versions smaller than the upper bound (automatically calculated as 1 plus the minor version) will be considered, right?

Correct, the boolean `freeze` was meant for the old GNOME versioning scheme, still used by some libraries, where the libraries broke API compatibility in a minor version bump. `glibmm` & co. come to mind as the worst offenders.